src/selectors/per-thread/markers.js

```diff
@@ -35,17 +32,17 @@ export type MarkerSelectorsPerThread = $ReturnType<
  * Create the selectors for a thread that have to do with either markers.
  */
 export function getMarkerSelectorsPerThread(threadSelectors: *) {
-  const _getMarkersTable: Selector<MarkersTable> = state =>
+  const _getMarkersTable: Selector<RawMarkerTable> = state =>
```

Comment:
nit: `_getMarkerTable` instead of `_getMarkersTable` (again, no `s`)?
Or even the more explicit `_getRawMarkerTable`?